new_01-vim-beginner.tutor

```diff
@@ -0,0 +1,1155 @@
+#             Welcome   to   the   VIM   Tutor
+
+Vim is a very powerful text editor that has many commands. Unlike most text
+editors where you use the keyboard to enter text and arrow keys to move your
+cursor, in Vim you use the keyboard to enter commands to manipulate text.
+These commands allow you to express complex thoughts in just a few keystrokes
+to move through text and manipulate it much faster than in most text editors.
+There is a great amount of commands, too many to explain in a tutor such as
+this, but as you will see over the course of this tutorial series these
+commands are not arbitrarily chosen, they form a sort of language with its own
+grammar.
+
+This tutor is designed to describe enough of the commands that you will be able
+to easily use Vim as an all-purpose editor.  It is important to remember that
+it is set up to teach by use. That means you need to do the exercises to learn
+them properly.  If you only read the text, you will soon forget what is most
+important!
+
+NOTE: The commands in the lessons will modify the text, but those changes won't
+be saved.  Don't worry about messing things up; just remember that pressing
+[«<Esc>»](<Esc>) and then [u](u) will undo the latest change.
```

Comment:
No, we aren't. The syntax for all these spans changed, we'll need to fix that before merging.